tests/test.bats

```diff
@@ -229,7 +234,8 @@ _healthcheck_wait ()
 
 	# Check Drush version
 	run docker exec -u docker "$NAME" bash -lc 'drush --version | grep "^ Drush Version   :  ${DRUSH_VERSION} $"'
-	[[ ${status} == 0 ]]
+        # Currently check on all other version than 7.4
+	[[ "${VERSION}" != "7.4" ]] && [[ ${status} == 0 ]]
```

Comment:
How is this passing now? If VERSION is 7.4 then this code will return `1` and should fail the test, but it does not.
cc @achekulaev

Reply:
@lmakarov i don't disable or turn off the installing of drush launcher just drush8.

Reply:
@sean-e-dietrich this check results in the following for PHP 7.4:

```
[[ "7.4" != "7.4" ]] && [[ 1 == 0 ]]
```

That exists with `1` and should fail the test, but that does not happen for whatever reason in bats, which does not make sense to me.

Doing the comparison in a sub-shell works as expected in bats:

```
( [[ "7.4" != "7.4" ]] && [[ 1 == 0 ]] )
```

It looks like the short form comparison in bats is equivalent to the long form:

```
if [[ "${VERSION}" != "7.4" ]]; then [[ ${status} == 0 ]]; fi
```

Though, logically (and in a regular bash script), they produce different results.

I pushed a fix for this.